### PR TITLE
Temporary fix for MD analysis when calculation is 'nscf' type.

### PR DESCRIPTION
**Pull request by mcbennet**
For calculations of 'nscf' type, the needed quantities for the MD analysis are not present in the output file.

### CODE REVIEW

**Comment by mcbennet** [comment]
All nxs-test tests pass.

**Comment by grahamlopez** [comment]
okay to test

File: nexus/lib/pwscf_analyzer.py [hunk 1]
```diff
@@ -140,35 +140,37 @@ def analyze(self):
...
-        md = obj()
-        for i,q in enumerate(quantities):
-            md[q] = md_res[i]
-        #end for
-        md.potential_energy = md.total_energy - md.kinetic_energy
-        self.md_data = md
-        self.md_stats = self.md_statistics()
-        if self.info.md_only:
-            return
+        if self.input.control.calculation != 'nscf':
```
Comment (jtkrogel) [inline nexus/lib/pwscf_analyzer.py]: Sorry, this was not meant to be left exposed.  Please place within a try-except block similar to the others below instead.

**Comment by mcbennet** [comment]
100% of tests pass with nxs-test.

**Pull request merged**